src/index.js

```diff
@@ -0,0 +1,61 @@
+// import _ from 'lodash';
+import './style.css';
+
+/*
+function component() {
+  const element = document.createElement('p');
+
+  // Lodash, now imported by this script
+  // element.innerHTML = _.join(['to-do', 'list'], ' ');
+  element.innerHTML = _.arr;
+
+  element.classList.add('p-toDoList');
+
+  return element;
+}
+
+document.body.appendChild(component());
+*/
+
+// build array to store objects
+
+const arr = [];
+const elementLu = document.querySelector('.to-do-list');
+
+// create object with properties (description=string,completed=boolean,index=number)
+// to be store in arr
+
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const task1 = new Task('Add correct style to awesome-book app', true, 1);
+arr.push(task1);
+
+const task2 = new Task('Add remove button to awesome-book app', false, 2);
+arr.push(task2);
+
+const task3 = new Task('Add contact section to awesome-book app', true, 3);
+arr.push(task3);
+
+// function to iterate the array and set the li items
+
+function createList(element) {
+  const elementLi = document.createElement('li');
+  const elementInput = document.createElement('input');
+  const elementLabel = document.createElement('label');
+
+  elementLu.appendChild(elementLi);
+  elementLi.appendChild(elementInput);
+  elementLi.appendChild(elementLabel);
+
+  elementInput.type = 'checkbox';
+
+  elementLabel.textContent = `${element.description}, ${element.completed}, ${element.index}`;
+}
```

Comment:
Consider styling the app to look similar to the original design, if you take a look at the video, you'd quickly see some icons to the right of the interface and also the clear completed button, but your version is lacking those. I recommend including them to create the basic set up for the functionality of the app.

Your design
![Screenshot](https://github.com/Zven94/to-do-list/assets/84930355/ac001466-7f0f-4f36-a9bd-f497ff205784)


Original design
![Screenshot (1)](https://github.com/Zven94/to-do-list/assets/84930355/214794bd-f9bf-4c83-9387-f46fb705e133)